src/internal/modules/charge-information/forms/start-date.js

```diff
@@ -44,13 +44,21 @@ const getDates = licence => {
   };
 };
 
-const minErrors = {
-  [MIN_LICENCE_START]: 'You must enter a date after the licence start date'
+const getDatesWithDocumentRoles = (licence, licenceDocumentsRoles) => {
+  const startDate = moment(licenceDocumentsRoles.startDate);
+  const minDate = startDate;
+  const isLicenceStart = startDate.isAfter(minDate);
+  return {
+    licenceStartDate: startDate,
+    minDate: isLicenceStart ? licenceDocumentsRoles.startDate : minDate.format(ISO_FORMAT),
+    minType: MIN_LICENCE_START,
+    maxDate: licence.endDate || '3000-01-01'
+  };
 };
 
 const getCommomCustomDateErrors = dates => ({
   'date.min': {
-    message: minErrors[dates.minType]
+    message: `You must enter a date after this date ${moment(dates.licenceStartDate).format('DD MMM YYYY')}`
```

Comment:
This error message is ambiguous. The user will not know what date it is I.e. licence start date or licence version start date or something else. Probably best to run this past UCD